KDIC/PlayerViewController.swift

```diff
@@ -6,17 +6,58 @@ class PlayerViewController: UIViewController {
   @IBOutlet weak var titleLabel: UILabel!
   @IBOutlet weak var subtitleLabel: UILabel!
   @IBOutlet weak var imageView: UIImageView!
-  
+
   @IBAction func togglePlayerButtonWasTapped(_ sender: Any) {
     KDICPlayer.toggle()
   }
+    
   override func viewDidLoad() {
     super.viewDidLoad()
+    
+    self.imageView.image = UIImage(named: "Equalizer")
+    
+    button.setImage(imgShell1, forState:.Normal);
+    button.setImage(coin, forState:.Highlighted);
+    
+    button.setBackgroundImage(resizeImageWithAspect(UIImage(data: data!)!,size:button.bounds.size), forState: .Normal);
+    button.contentMode = UIViewContentMode.ScaleAspectFit;
+    button.clipsToBounds = true;
+    
   }
+    
   override func viewDidAppear(_ animated: Bool) {
     super.viewDidAppear(animated)
     print("About to play")
     KDICPlayer.play()
     print("Playing!")
   }
+    
+    func resizeImageWithAspect(image: UIImage,size: CGSize)->UIImage
```

Comment:
I don't think you need this. You can use the `Aspect Fill` setting I showed you on the storyboard to adjust how the image appears in the `UIImage` view.